homeassistant/components/mqtt/mixins.py

```diff
@@ -269,8 +283,15 @@ async def async_discover(discovery_payload: MQTTDiscoveryPayload) -> None:
         try:
             config: DiscoveryInfoType = discovery_schema(discovery_payload)
             await async_setup(config, discovery_data=discovery_data)
+        except vol.MultipleInvalid as err:
```

Comment:
Why do we only catch `MultipleInvalid`, shouldn't we catch the voluptuous' base exception `vol.Invalid` instead?

Reply:
I'll Add that one too